tools/softhsm/extensions-ecdsa-pk8.pem

```diff
@@ -0,0 +1,6 @@
+-----BEGIN PRIVATE KEY-----
```

Comment:
this is the result of `openssl pkcs8 -topk8 -nocrypt -in extensions-ecdsa.pem -out extensions-ecdsa-pk8.pem` where extensions-ecdsa.pem is the private key field from the signer `autograph.yaml`